clientd3d/web.h

```diff
@@ -39,7 +39,8 @@ enum AccountStatusWebResponse
     FAILURE = 1 << 2
 };
 
-void WebLaunchBrowser(char *url);
+void WebLaunchBrowser(const char *url);
+bool ContainsSpaces(const std::string& str);
```

Comment:
This doesn't need to be in the header; we don't want other files calling ContainsSpaces.  Move it to the .c file and make it static.

Reply:
Understood.  Made static and moved to before its WebLaunchBrowser caller.  Removed from header file.